test/50-assert-15-assert_output.bats

```diff
@@ -26,9 +26,28 @@ load test_helper
   [ "${lines[3]}" == '--' ]
 }
 
-@test 'assert_output(): reads <expected> from STDIN' {
+@test 'assert_output(): succeeds if output is non-empty' {
   run echo 'a'
-  run assert_output <<STDIN
+  run assert_output
+echo "$output"
```

Comment:
I think you may have left this in by accident. I do this a lot too.. intentionally not identing the `echo`s so I will spot them before commiting, but I end up leaving them there. :smile: